iterable-extension/src/main/java/com/mparticle/ext/iterable/IterableExtension.java

```diff
@@ -77,15 +77,18 @@ private void processPushOpens(EventProcessingRequest processingRequest) throws I
                     Map<String, Object> payload = mapper.readValue(event.getPayload(), Map.class);
                     if (payload.containsKey("itbl")) {
                         //Android and iOS have differently encoded payload formats. See the tests for examples.
+                        Map<String, Object> iterableMap;
```

Comment:
I'd put the declaration within the same block where it's initialized – to make sure it can never become `null` if we change the code later.